libobs/obs-output.c

```diff
@@ -1829,9 +1829,9 @@ static bool get_audio_and_video_packets(struct obs_output *output,
 
 static bool initialize_interleaved_packets(struct obs_output *output)
 {
-	struct encoder_packet *video[MAX_OUTPUT_VIDEO_ENCODERS];
-	struct encoder_packet *audio[MAX_OUTPUT_AUDIO_ENCODERS];
-	struct encoder_packet *last_audio[MAX_OUTPUT_AUDIO_ENCODERS];
+	struct encoder_packet *video[MAX_OUTPUT_VIDEO_ENCODERS] = {NULL};
+	struct encoder_packet *audio[MAX_OUTPUT_AUDIO_ENCODERS] = {NULL};
+	struct encoder_packet *last_audio[MAX_OUTPUT_AUDIO_ENCODERS] = {NULL};
```

Comment:
We typically use `var = {0}` to zero-initialize arrays and structs in OBS code, functionally it makes no difference but it might be nice for consistency.